contracts/misc/ERC721OracleWrapper.sol

```diff
@@ -34,11 +35,13 @@ contract ERC721OracleWrapper is IEACAggregatorProxy {
     constructor(
         address _provider,
         address _oracleAddress,
-        address _asset
+        address _asset,
+        uint256 _expirationPeriod
```

Comment:
I think it's hard to sync expirationPeriod between OracleWrapper & NFTFloorOracle, it's better to do it inside getTwap

Reply:
set in OracleWrapper allow us to set different `expirationPeriod` for each asset, and we can set to same value when deploy and change it as we want later, it is a configurable param.

Reply:
@yrong but image that we changed `expirationPeriod` in NFTFloorOracle and we didn't update here. Will that cause issues?

Reply:
we can prepare a script for change both values if necessary, and I do think separate to 2 config could be helpful. different use case one for global and one for each asset.